### ISSUE TITLE
sandbox URL

### ISSUE THREAD

**Issue by gedw99** [jtrotsky/wise-cli#2]
Hey

Currently the URL is hardcoded to this in the code.
// DefaultAPIBaseURL is the default base URL for API requests
const DefaultAPIBaseURL = "https://api.transferwise.com"

Would it be useful to also be able to tell it to use the sandbox URL ?
https://api.sandbox.transferwise.tech/

Maybe best way is to have a config boolean called "useSandbox", and then in pkg/client.go pick the right URL.

--- 

Also i noticed that there use different versions in the URL depending on the operation.
https://api-docs.transferwise.com/#exchange-rates-list
- GET https://api.sandbox.transferwise.tech/v1/rates

https://api-docs.transferwise.com/#quotes-create
- POST https://api.sandbox.transferwise.tech/v2/quotes

saw that your using the old one "/v1/quotes/" for some reason. Maybe the Sandbox and Production use different versions ?

**Comment by jtrotsky** [comment]
Good idea @gedw99 , sorry I took ages to see this. Have upgraded to v2 quotes

**Comment by gedw99** [comment]
Thanks @jtrotsky.
